modules/serverless-sinkbinding-yaml.adoc

```diff
@@ -3,9 +3,9 @@
 // serverless/event_workflows/serverless-sinkbinding.adoc
 
 [id="serverless-sinkbinding-yaml_{context}"]
-= Using SinkBinding with the YAML method
+= Using sink binding with the YAML method
```

Comment:
I'll leave all of these as they are for now because we just use "sink binding" elsewhere: https://knative.dev/docs/eventing/sources/sinkbinding/
There is "a" `SinkBinding` resource, but I think when it's referred to as just sink binding it's more talking about the concept than a specific thing.
Can revisit maybe later.